quicfire_tools/inputs.py

```diff
@@ -722,3 +745,356 @@ def from_file(cls, directory: str | Path):
             maximum_firebrand_ignitions=maximum_firebrand_ignitions,
             minimum_landing_angle=minimum_landing_angle,
             maximum_firebrand_thickness=maximum_firebrand_thickness)
+
+
+class QUIC_fire(InputFile):
+    """
+    Class representing the QUIC_fire.inp input file. This file
+    contains the parameters relating to the fire simulation and
+    outputs.
+
+    Parameters
+    ----------
+    nz : int
+        Number of fire grid cells in the z-direction.
+    output_times: int | OutputTimes
+        @anthony
+    time_now : int
+        When the fire is ignited in Unix Epoch time (integer seconds since
+        1970/1/1 00:00:00). Must be greater or equal to the time of the first
+        wind
+    sim_time : int
+        Total simulation time for the fire [s]
+    fire_flag : int
+        Fire flag, 1 = run fire; 0 = no fire
+    random_seed : int
+        Random number generator, -1: use time and date, any other integer > 0
+        is used as the seed
+    fire_time_step : int
+        time step for the fire simulation [s]
+    quic_time_step : int
+        Number of fire time steps done before updating the quic wind field
+        (integer, >= 1)
+    stretch_grid_flag : int
+        Vertical stretching flag: 0 = uniform dz, 1 = custom
+    file_path : str
```

Comment:
Do you think that it is worth breaking these into individual file paths? I'm not sure how QUIC-Fire handles this. Let me know if you know.

Reply:
Not sure what you're referring to here. Stretch grid flag or file path...? Honestly we should probably not make the file path a user input and just have it be the convention that .dat files are with all the other inputs